data/ecosystem.ts

```diff
@@ -1,2466 +1,2409 @@
 /* eslint-disable sonarjs/no-duplicate-string */
-import type { Network } from "../src/models/company";
+import type { Network } from '../src/models/company'
```

Comment:
Thanks but in this project, we are using " instead of '
And we put ";" at the end of each line in interface declaration scope.

I've just pushed a PR https://github.com/419Labs/starknet-ecosystem.com/pull/230 to include the data directory when running the linter